app/controllers/groups_controller.rb

```diff
@@ -25,6 +25,10 @@ def create
 
   def show
     @group_type = @group.group_type
+    if @group_type == "mentor"
```

Comment:
Quick Q about this code. Wondering if it wouldn't be easier to use the line of code that Halima had written in the GroupUser controller and repurpose it: @group_user = GroupUser.create(user_id: current_user.id, group_id:params[:group].to_i)